tests/mypy/outputs/1.2.0/mypy-plugin-strict_ini/plugin_fail_baseConfig.py

```diff
@@ -0,0 +1,366 @@
+from typing import Any, Generic, List, Optional, Set, TypeVar, Union
+
+from pydantic import BaseModel, Extra, Field, field_validator
+from pydantic.dataclasses import dataclass
+
+
+# placeholder for removed line
```

Comment:
What is the removed line here?

Reply:
That was necessary when we had line numbers everywhere or else you'd get a huge diff. That's no longer the case so I will remove it, but let me make sure everything passses first